echoregions/lines/lines.py

```diff
@@ -33,45 +37,19 @@ def __getitem__(self, idx: int) -> Union[Dict, List]:
         """Indexing lines object will return the point at that index"""
         return self.points[idx]
 
-    @property
-    def output_file(self) -> Union[List[str], str]:
-        """Path(s) to the list of files saved.
-        String if a single file. List of strings if multiple.
-        """
-        return self._parser.output_file
-
-    @property
-    def input_file(self) -> str:
-        """String path to the EVL file"""
-        return self._parser.input_file
-
     @property
     def nan_depth_value(self) -> Union[int, float]:
         return self._nan_depth_value
 
+    @property
+    def data(self) -> DataFrame:
+        return self._data
+
```

Comment:
I don't see a lot of value in keeping `nan_depth_value` as a property -- what do you think about removing it? Also curious why you decided to have `.data` as a way to access `._data` -- is it for potential future changes?